src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: #fff;
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin-top: 0;
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__code {
+    margin-bottom: 6px;
+    margin-top: 0;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: #616070;
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__review {
+    text-align: right;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 15px;
+    color: #060b35;
+    margin-top: 11px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__text {
+    font-size: 12px;
+    line-height: 15px;
+    font-weight: 400;
+    color: #616070;
+  }
+
+  &__amount {
+    font-size: 16px;
+    line-height: 14px;
+    font-weight: 700;
+    color: #060b35;
+  }
+
+  &__buy {
+    text-decoration: none;
+    color: rgba(255, 255, 255, 1);
+    padding: 12px 69px 12px 70px;
+    background: var(--color);
+    border-radius: 5px;
+    font-weight: 700;
+    line-height: 16px;
+    font-size: 14px;
+    text-transform: uppercase;
+    border: 1px solid transparent;
+    transition: all 1s linear;
+
+    &:hover {
+      color: var(--color);
+      background-color: #fff;
+      border-color: var(--color);
+    }
+  }
+}
+
+html {
+  font-family: "Roboto", Helvetica, sans-serif;
+  --color: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
you should remove this styles from card.scss file, they're not related to the card